v03_pipeline/lib/reference_datasets/reference_dataset.py

```diff
@@ -349,7 +362,12 @@ def get_ht(
     },
 }
 CONFIG[ReferenceDatasetQuery.clinvar_path_variants] = CONFIG[ReferenceDataset.clinvar]
+CONFIG[ReferenceDatasetQuery.clinvar_path_variants].EXCLUDE_FROM_ANNOTATIONS = True
```

Comment:
What if we added `ReferenceDatasetQuery.clinvar_path_variants` to the config above and did:

```
  ReferenceDatasetQuery.clinvar_path_variants: {
     EXCLUDE_FROM_ANNOTATIONS: True,
     **CONFIG[ReferenceDataset.clinvar],
  }
```

I _think_ that works? (it works in js at least!)